recipes/bolt/meta.yaml

```diff
@@ -0,0 +1,119 @@
+{% set version = "17.0.6" %}
+{% set major_ver = version.split(".")[0] %}
+
+# as of LLVM 17, we expect an "rc" suffix for the release candidates
+{% set extra = "rc" if version.split(".")[-1] not in "0123456789" else "" %}
+{% set extra = "git" if version.split(".")[-1] == "dev0" else extra %}
+
+package:
+  name: bolt-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-project-{{ version.replace(".rc", "rc") }}.src.tar.xz
+  sha256: 58a8818c60e6627064f312dbf46c02d9949956558340938b71cf731ad8bc0813
+  # matching https://github.com/conda-forge/llvmdev-feedstock/tree/main/recipe/patches,
+  # since we rebuild llvmdev here (until https://github.com/llvm/llvm-project/pull/97130)
+  patches:
+    - patches/0001-pass-through-QEMU_LD_PREFIX-SDKROOT.patch
+    # backports patch for issues with LLJIT, see
+    # https://github.com/llvm/llvm-project/commit/122ebe3b500190b1f408e2e6db753853e297ba28
+    - patches/0002-ORC-Use-EPC-bootstrap-symbols-to-communicate-eh-fram.patch
+
+build:
+  number: 0
+  merge_build_host: false
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - libcxx {{ cxx_compiler_version }}  # [osx]
+  host:
+    - libcxx {{ cxx_compiler_version }}  # [osx]
+    # needs aarch/ppc/arm (re)build of conda-forge/backtrace-feedstock
+    - backtrace                          # [unix and x86]
+    - libxml2
+    - zlib
+    - zstd
+
+outputs:
+  # Contains bolt support libraries
+  - name: libbolt
+    script: install_bolt.sh   # [unix]
+    script: install_bolt.bat  # [win]
+    build:
+      activate_in_script: true
+    requirements:
+      build:
+        - {{ stdlib('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - ninja
+        - libcxx {{ cxx_compiler_version }}  # [osx]
+      run_constrained:
+        - llvm        {{ version }}
+        - llvmdev     {{ version }}
+        - clang       {{ version }}
+        - clang-tools {{ version }}
+    test:
+      commands:
+      {% for each_lib in [
+          "Core", "Passes", "Profile", "Rewrite", "RuntimeLibs",
+          "TargetAArch64", "TargetX86", "Utils"
+      ] %}
+        - test -f $PREFIX/lib/libLLVMBOLT{{ each_lib }}.a               # [unix]
+        - if not exist %LIBRARY_LIB%\LLVMBOLT{{ each_lib }}.lib exit 1  # [win]
+      {% endfor %}
+        # only on linux-64
+        - test -f $PREFIX/lib/libbolt_rt_hugify.a                       # [linux64]
+        - test -f $PREFIX/lib/libbolt_rt_instr.a                        # [linux64]
```

Comment:
Also check that the cmake files are here

Reply:
Only way to do that currently is to clobber
```
lib/cmake/llvm/LLVM-Config.cmake
lib/cmake/llvm/LLVMConfig.cmake
lib/cmake/llvm/LLVMConfigVersion.cmake
lib/cmake/llvm/LLVMExports-release.cmake
lib/cmake/llvm/LLVMExports.cmake
```
from llvmdev, unfortunately. But I guess better than nothing.

Reply:
bolt doesn't have different cmake files?

Reply:
nope, it's built as part of llvm. I guess we could ask for separate CMake files in the "standalone" upstream PR...

Reply:
Does this change your opinion somehow, or are you fine with the `always_include_files:` (https://github.com/conda-forge/staged-recipes/pull/22204/commits/0e969618dca76f8d5655347cf552e1642133a05f)?

Reply:
So I've got the CMake metadata working now, but OTOH, it'll be broken unless we let `libbolt-devel` depend on `llvmdev`.

Reply:
@isuruf, could you let me know how you'd like to proceed with the CMake metadata here?

Reply:
Let's make `libbolt-devel` depend on `llvmdev`

Reply:
Thanks. I'm planning to merge this once CI passes (well, on unix at least; the rest I'll fix on the feedstock).

Reply:
Please don't. We require reviews from everyone including core team.

Reply:
Then please don't drip-feed change requests. If there's been two rounds of review and no more open comments are left, it's not unreasonable to think that things are wrapped up.